tests/main/ubuntu-core-classic/task.yaml

```diff
@@ -0,0 +1,40 @@
+summary: Ensure classic dimension works correctly
+systems: [ubuntu-core-16-64]
+environment:
+    SUDO_USER: test
+prepare: |
+    echo "test ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/create-test
+restore: |
+    rm -f /etc/sudoers.d/create-test
```

Comment:
Thanks! On classic we automatically snap remove all snaps that are not initially installed, so this cleanup happens automatically.